### ISSUE TITLE
Breaks repeat

### ISSUE THREAD

**Issue by dag** [paradigm/TextObjectify#7]
Say I have

```
Here's an <em>example</em> for you!
```

and then I do `/em<CR>ciwstrong<Esc>n.`.

Without TextObjectify, I end up with

```
Here's an <strong>example</strong> for you!
```

which is what I want, but with TextObjectify I get

```
trong
```

That's the whole line.

**Comment by paradigm** [comment]
Sorry it took me so long to get back to you.  Yep, I see that here as well.  It's actually been really bugging me while I'm doing my own work.  I'll see if I can fix this and the other hanging TOify issues in the near future.

I think the cleanest way to remedy this will be to utilize https://github.com/tpope/vim-repeat

**Comment by justinmk** [comment]
Well, I'm dumb. That last commit didn't actually fix #7, it just fixed a regression that I caused. Repeat still doesn't work. But at least there are no regressions (?) :)

**Comment by bruno-** [comment]
Any progress on this?
This plugin got me really excited, but now I'm kinda "low" that I see there's this bug..

**Comment by wellle** [comment]
@bruno- Check out [targets.vim](https://github.com/wellle/targets.vim) for similar functionality with working repeatability.

@paradigm Feel free to delete this comment if you don't want this kind of cross promotion.

**Comment by bruno-** [comment]
@wellle, you mentioned it in a reddit discussion a while ago. I have it installed since, working flawless - thanks a lot for building it.

**Comment by paradigm** [comment]
Yes, my apologies, for the time being I'd recommend targets.vim instead.  While I'd love to revisit this, it doesn't look like it's going to happen in a timely manner.

**Comment by ChanderG** [comment]
I have managed to integrate `repeat.vim` [here](https://github.com/ChanderG/TextObjectify/commit/5428e590fbcdf8de028954702f88effa17d01b7f). I can open a PR if needed.
